indexers/base-sepolia.yaml

```diff
@@ -49,9 +65,31 @@ dataSources:
       file: ./dist/index.js
       handlers:
         - kind: ethereum/LogHandler
-          handler: handleBaseSepoliaTransferEvent
+          handler: handleTransferEvent
           filter:
             topics:
               - "Transfer(address indexed from, address indexed to, uint256 amount)"
 
+  # EthereumHost contract
+  - kind: ethereum/Runtime
```

Comment:
We don't need to index the handler, all the events have been moved to the host in the latest code